content/en/guides/hosting/iam/authentication/service-accounts.md

```diff
@@ -59,4 +75,68 @@ A team-scoped service account cannot log runs to a [team or restricted-scoped pr
 
 ### External service accounts
 
-In addition to **Built-in** service accounts, W&B also supports team-scoped **External service accounts** with the W&B SDK and CLI using [Identity federation]({{< relref "./identity_federation.md#external-service-accounts" >}}) with identity providers (IdPs) that can issue JSON Web Tokens (JWTs).
\ No newline at end of file
+In addition to **Built-in** service accounts, W&B also supports team-scoped **External service accounts** with the W&B SDK and CLI using [Identity federation]({{< relref "./identity_federation.md#external-service-accounts" >}}) with identity providers (IdPs) that can issue JSON Web Tokens (JWTs).
+
+## Best practices
```

Comment:
I think you can combine Security and Operational best practices into a single list.

The contents within "Common use cases" would be better moved up into the overview of service accounts and does not need a heading.
Troubleshooting should be an H2.